src/components/Footer/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react';
+import { Box, Text, Footer, ResponsiveContext, Button } from 'grommet';
+
+// import Footer from "./styles";
+export const AppFooter = () => {
+  const size = useContext(ResponsiveContext);
+
+  const footerLinks = [
+    { label: 'Terms' },
```

Comment:
Im guessing we will need pass `href` to these links..

also these names are not the same as on figma. 

Fix these to be:
Privacy Policy 
Cookies 
Terms of Use
Do Not Sell My Personal Information